test/users/jglewis/wrong_error_message_april_26_2010.future

```diff
@@ -0,0 +1,9 @@
+bug: cast to non-type is not flagged as an error
+
+The cast on lines 18-20 is casting to the void value, not type.  It currently
+compiles and runs, but should be an error.
+
+This is the same basic problem as shown by the tests:
+  expressions/vass/cast-to-non-type-1.future
+  expressions/vass/cast-to-non-type-2.future
+
```

Comment:
This makes sense to me.  I'm always surprised that our compiler still lets this case slip through... it seems like it should be a pretty easy hole to fix.  Would you file an issue against it assuming we don't have one already? (and add it to the 1.15.0 milestone in hopes that we'll finally plug the hole?  :) ).